common/static/common/js/discussion/views/thread_response_view.js

```diff
@@ -181,6 +182,18 @@
                 return view;
             };
 
+            ThreadResponseView.prototype.toggleCommentSubmitButton = function(event) {
+                var id = event.target.id.split('-').slice(-1)[0];
+                var query_selector = 'form[data-id="'+ id +'"] .discussion-submit-comment';
+                var comment_button = document.querySelector(query_selector);
+                if (event.target.value.length > 0) {
+                    comment_button.disabled = false;
+                }
+                else {
+                    comment_button.disabled = true;
```

Comment:
modify this piece of code according to my suggestion https://github.com/edly-io/edx-platform/pull/29#discussion_r354274483

Reply:
Done